src/test/java/com/cgdpd/library/service/BookServiceTest.java

```diff
@@ -96,6 +110,103 @@ public void should_throw_exception_if_author_not_exist() {
     }
 
     @Test
+    void should_return_books_with_pagination() {
+        // given
+        var searchCriteria = SearchBookCriteria.builder().build();
+        Specification<BookEntity> spec = BookSpecifications.byBookSearchCriteria(searchCriteria);
+        int pageNumber = 0;
+        int pageSize = 10;
+        var sortParam = new SortParam("title", SortParam.Direction.ASC);
+        var paginationCriteria = new PaginationCriteria(pageNumber, pageSize,
```

Comment:
Use builder, and don't create the variables, just use the values directly in the builder